turbinia/workers/__init__.py

```diff
@@ -41,9 +41,21 @@
 from turbinia import TurbiniaException
 from turbinia import log_and_report
 from turbinia.lib import docker_manager
+from prometheus_client import Gauge
 
 log = logging.getLogger('turbinia')
 
+turbinia_worker_started_tasks_total = Gauge(
+    'turbinia_worker_started_tasks_total',
+    'Total number of started worker tasks')
+turbinia_worker_tasks_completed_total = Gauge(
+    'turbinia_worker_tasks_completed_total',
+    'Total number of completed worker tasks')
+turbinia_worker_queued_tasks_total = Gauge(
```

Comment:
Should we use `turbinia_worker_tasks_queued_total` (or switch the other variables to match this one) so the word ordering is consistent?